components/onboarding-modal/OnboardingContentBox.js

```diff
@@ -0,0 +1,165 @@
+  static propTypes = {
+    step: PropTypes.number,
+    collective: PropTypes.object,
+    LoggedInUser: PropTypes.object,
+    addAdmins: PropTypes.func,
+    addContact: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      admins: [],
+    };
+  }
+
+  componentDidMount() {
+    const member = this.props.LoggedInUser.memberOf.filter(member => member.collective.id === this.props.collective.id);
+    this.setState({
+      admins: [{ role: 'ADMIN', member: this.props.LoggedInUser.collective, id: member[0].id }],
+    });
+  }
+
+  render() {
+    const { step, collective, LoggedInUser, addAdmins, addContact } = this.props;
+    const { admins } = this.state;
+
+    return (
+      <Container display="flex" flexDirection="column" width="80%" alignItems="center">
+        {step === 0 && (
+          <Flex maxWidth={['336px']}>
+            <H1
+              fontSize={['H5']}
+              lineHeight={['H5']}
+              fontWeight="bold"
+              color="black.900"
+              textAlign="center"
+              mb={[2, 4]}
+              mx={[2, null]}
+            >
+              The {collective.name} Collective has been created! 🎉
+            </H1>
+          </Flex>
+        )}
+        {step === 1 && (
+          <Fragment>
+            <Flex maxWidth={['336px']}>
+              <H1 fontSize={['H5']} lineHeight={['H5']} fontWeight="bold" color="black.900" textAlign="center" mb={4}>
+                Add administrators
+              </H1>
+            </Flex>
+            <Flex px={3} width="100%">
+              <P my={2} fontSize="Caption" textTransform="uppercase" color="black.700">
+                Administrators
+              </P>
+              <Flex flexGrow={1} alignItems="center">
+                <StyledHr width="100%" ml={2} />
+              </Flex>
+            </Flex>
+            {admins.length > 0 && (
+              <Flex px={3} width="100%" flexWrap="wrap">
+                {admins.map(admin => (
+                  <OnboardingProfileCard
+                    key={admin.member.id}
+                    user={admin.member}
+                    adminUser={LoggedInUser.collective}
+                  />
+                ))}
+              </Flex>
+            )}
+            <Flex px={3} width="100%">
+              <P my={2} fontSize="Caption" textTransform="uppercase" color="black.700">
+                Invite administrators
+              </P>
+              <Flex flexGrow={1} alignItems="center">
+                <StyledHr width="100%" ml={2} />
+              </Flex>
+            </Flex>
+
+            <Flex my={2} px={3} flexDirection="column" width="100%">
+              <CollectivePickerAsync
+                creatable
+                types={['USER']}
+                onChange={option => {
+                  // only assign admins if they are not in the list already
+                  const duplicates = admins.filter(admin => admin.member.id === option.value.id);
```

Comment:
**[:ice_cream: Details & personal preferences]** We don't need to go through the whole list

```suggestion
                  const hasDuplicates = admins.some(admin => admin.member.id === option.value.id);
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
I found that this didn't work the same way and the duplicate admin cards were still displayed.

Reply:
Did you also update the code below to remove the `.length` like below? It's strange, but anyway that's a very minor detail.

```es6
admins: hasDuplicates ? admins : [...state.admins, { role: 'ADMIN', member: option.value }]
```